--- xml-xalan/src/xalanc/Utils/Makefile.in~	2009-09-23 00:58:28.000000000 +0300
+++ xml-xalan/src/xalanc/Utils/Makefile.in	2009-09-23 00:58:31.692161855 +0300
@@ -99,9 +99,9 @@
 locale:  $(XSL_NLS_DIR)/icu/$(LOCALE).txt $(MESSAGELIB)
 
 $(MESSAGELIB) : $(XSL_NLS_DIR)/icu/$(LOCALE).txt
-	$(GENRB) -p $(LOC_PROJECT_NAME) -d $(XSL_NLS_DIR)/icu -i $(XSL_NLS_DIR)/icu $(XSL_NLS_DIR)/icu/$(LOCALE).txt
-	echo $(XSL_NLS_DIR)/icu/$(LOC_PROJECT_NAME)_$(LOCALE).res > $(XSL_NLS_DIR)/icu/res-file-list.txt
-	$(PKGDATA) --name $(LOC_PROJECT_NAME) -T $(XSL_NLS_DIR)/icu -v -O $(ICUROOT)/lib/icu/Makefile.inc  --mode dll -d $(XSL_LIB_DIR) $(XSL_NLS_DIR)/icu/res-file-list.txt -M "prefix=$(ICUROOT) TARGET=$(LOC_LIB) BIR_LDFLAGS=\"-Bsymbolic -Wl,-soname,$(LOC_LIBNAME)$(SHLIBSUFFIX).$(LIB_MAJOR_VER)\""
+	$(GENRB) -d $(XSL_NLS_DIR)/icu -i $(XSL_NLS_DIR)/icu $(XSL_NLS_DIR)/icu/$(LOCALE).txt
+	echo $(XSL_NLS_DIR)/icu/$(LOCALE).res > $(XSL_NLS_DIR)/icu/res-file-list.txt
+	$(PKGDATA) --name $(LOC_PROJECT_NAME) -T $(XSL_NLS_DIR)/icu -v -O $(ICUROOT)/lib/icu/pkgdata.inc --mode dll -d $(XSL_LIB_DIR) $(XSL_NLS_DIR)/icu/res-file-list.txt -r $(LIB_MAJOR_VER).0
 	$(LN) -fs $(LOC_LIB) $(XSL_LIB_DIR)/$(LOC_LIBNAME)$(SHLIBSUFFIX)
 ifeq ($(PLATFORM), AIX)
 	$(LN) -fs $(LOC_LIB) $(XSL_LIB_DIR)/$(LOC_LIBNAME)$(LIB_MAJOR_VER)$(SHLIBSUFFIX)
