--- xml-xalan/c/src/xalanc/Utils/Makefile.in.orig	2005-08-16 03:06:47.000000000 +0200
+++ xml-xalan/c/src/xalanc/Utils/Makefile.in	2007-01-22 15:02:37.618134000 +0100
@@ -101,7 +101,7 @@
 $(MESSAGELIB) : $(XSL_NLS_DIR)/icu/$(LOCALE).txt
 	$(GENRB) -p $(LOC_PROJECT_NAME) -d $(XSL_NLS_DIR)/icu -i $(XSL_NLS_DIR)/icu $(XSL_NLS_DIR)/icu/$(LOCALE).txt
 	echo $(XSL_NLS_DIR)/icu/$(LOC_PROJECT_NAME)_$(LOCALE).res > $(XSL_NLS_DIR)/icu/res-file-list.txt
-	$(PKGDATA) --name $(LOC_PROJECT_NAME) -T $(XSL_NLS_DIR)/icu -v -O $(ICUROOT)/lib/icu/Makefile.inc  --mode dll -d $(XSL_LIB_DIR) $(XSL_NLS_DIR)/icu/res-file-list.txt -M "prefix=$(ICUROOT) TARGET=$(LOC_LIB)"   
+	$(PKGDATA) --name $(LOC_PROJECT_NAME) -T $(XSL_NLS_DIR)/icu -v -O $(ICUROOT)/lib/icu/Makefile.inc  --mode dll -d $(XSL_LIB_DIR) $(XSL_NLS_DIR)/icu/res-file-list.txt -M "prefix=$(ICUROOT) TARGET=$(LOC_LIB) BIR_LDFLAGS=\"-Bsymbolic -Wl,-soname,$(LOC_LIBNAME)$(SHLIBSUFFIX).$(LIB_MAJOR_VER)\""
 	$(LN) -fs $(LOC_LIB) $(XSL_LIB_DIR)/$(LOC_LIBNAME)$(SHLIBSUFFIX)
 ifeq ($(PLATFORM), AIX)
 	$(LN) -fs $(LOC_LIB) $(XSL_LIB_DIR)/$(LOC_LIBNAME)$(LIB_MAJOR_VER)$(SHLIBSUFFIX)
