--- xml-xalan/c/runConfigure~	2007-01-16 02:51:41.542772924 +0200
+++ xml-xalan/c/runConfigure	2007-01-16 02:51:52.393015979 +0200
@@ -198,14 +198,15 @@
 
 # Get the command line parameters
 if test -x /bin/getopt -o -x /usr/bin/getopt; then
-    set -- `getopt C:p:P:c:x:dm:n:t:m:M:r:b:l:z:h $*`
-
+	t=`getopt -o C:p:P:c:x:dm:n:t:m:M:r:b:l:z:h -- "$@"`
     if [ $? != 0 ]
         then
         usage
         exit ${ERROR_EXIT_CODE}
     fi
 
+    eval set -- "$t"
+
 while [ $# -gt 0 ]
    do
    case $1 in
@@ -358,7 +359,7 @@
     elif test $platform = "os390"; then
         debugflag="-DNDEBUG";
     elif test $platform = "linux"; then
-		if test $cppcompiler = "icpc"; then
+		if test "$cppcompiler" = "icpc"; then
 			debugflag="-O3 -DNDEBUG";
 		else
 			debugflag="-O2 -DNDEBUG";
